src/index.js

```diff
@@ -6,7 +6,7 @@ import thunk from 'redux-thunk';
 import registerServiceWorker from './registerServiceWorker';
 
 import './index.css';
-import App from './App';
+import { ConnectedApp } from './App';
```

Comment:
Very common convention is to export connected component as default and "clear" component as named import. So the previous version would actually be more often seen.